### ISSUE TITLE
Use the Cocoapods spec CDN in the Podfile

### ISSUE THREAD

**Issue by NSExceptional** [PopcornTimeTV/PopcornTimeTV#907]
In the podfile, you have this line:

```ruby
source 'https://github.com/CocoaPods/Specs'
```

Change it to this:
```ruby
source 'https://cdn.cocoapods.org/'
```

Cocoapods has migrated away from hosting the spec repo on GitHub; [they now use a CDN](https://blog.cocoapods.org/CocoaPods-1.7.2/) which is much faster for cloning / updating.

**Comment by f-bone** [comment]
what about podfile.lock? there is also an outdated URL regarding SPECS.. should we change that too?

**Comment by NSExceptional** [comment]
No, it will change after you run `pod install` after changing `Podfile`

**Comment by f-bone** [comment]
yes I read about it on some site! thanks man - total noob here..
